themes/wikipedia/wikipedia.user.css

```
@@ -368,6 +368,7 @@
 @-moz-document domain("wikipedia.org"), domain("wikimedia.org") {
     /* -=- Logo Replacement -=- */
     #p-logo a {
+        background-size: 135px auto;
```

This should ensure that the logo is always the right size regardless of the navigation bar's size:

```suggestion
        background-size: contain;
```